docs/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+
+
+Maintainers will complete the following section:
+- [ ] Commit messages are descriptive enough
+- [ ] "Signed-off-by:" line is present in each commit
+- [ ] Unit tests pass
```

Comment:
Do we need this checkbox when we use travis-CI integration?

Reply:
Good point, no. Removed.